crypto/ledger/ledger_test.go

```diff
@@ -192,6 +192,7 @@ func getFakeTx(accountNumber uint32) []byte {
 }
 
 func TestSignaturesHD(t *testing.T) {
```

Comment:
The test seems to be passing, we can just enable it.

Reply:
Right. When I run this individually, it works, on within its package. It fails for me when running the whole tests suite.
Enabled for the moment, we will figure once the CI is fully enabled (https://polygon.atlassian.net/browse/POS-2725)